docs-beta/static/img/Wave Shape 1.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="1920" height="256" viewBox="0 0 1920 256" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
To allow it to stretch try adding preserveAspectRatio="none" into the top of the svg:

<svg preserveAspectRatio="none" width="1920" height="256"

Reply:
https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/preserveAspectRatio